org.kde.isoimagewriter.json

```diff
@@ -0,0 +1,45 @@
+{
+    "id": "org.kde.isoimagewriter",
+    "rename-icon": "isoimagewriter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "isoimagewriter",
+    "rename-icon": "isoimagewriter",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=all",
+        "--filesystem=host",
```

Comment:
is host access actually needed?

Reply:
shouldn't `/media` and `/run/media` be enough to access the removable medias to write?

Reply:
At the moment, this is using this polkit helper: https://invent.kde.org/utilities/isoimagewriter/-/blob/master/isoimagewriter/imagewriter_helper.cpp

This is used to write the file (given by path, not by fd as the portals like) into the device.
https://invent.kde.org/utilities/isoimagewriter/-/blob/master/isoimagewriter/imagewriter.cpp#L65

I am not sure how any of this is supposed to work on flatpak without changing really...